_includes/doc/admin-guide/options/channel-args.md

```diff
@@ -0,0 +1,15 @@
+## channel-args()
+
+|  Type:|      |
+|Default:| -|
+
+*Description:* The `channel-args()` option is available in gRPC-based drivers. The option accepts name-value pairs and sets channel arguments defined in the [GRPC Core library documentation](https://grpc.github.io/grpc/core/group__grpc__arg__keys.html).
```

Comment:
use the external_links.yml for external links plz (here, and everywhere bellow as well)